homeassistant/components/zwave_js/api.py

```diff
@@ -390,6 +391,7 @@ async def websocket_network_status(
             "suc_node_id": controller.suc_node_id,
             "supports_timers": controller.supports_timers,
             "is_heal_network_active": controller.is_heal_network_active,
+            "inclusion_state": controller.inclusion_state,
```

Comment:
For future readers, the value is an IntEnum:
https://github.com/home-assistant-libs/zwave-js-server-python/blob/47124d7670cfbff7f5edc0f002579efa67dc1eb2/zwave_js_server/const/__init__.py#L269-L277

Reply:
should I explicitly use `controller.inclusion_state.value` to make it more clear?

Reply:
I don't think it matters. It will be converted to a number by the json serializer before the websocket consumer gets it. It's the possible values that may be interesting and those aren't shown anyway by the code here.